pennylane_lightning_kokkos/lightning_kokkos.py

```diff
@@ -68,10 +69,11 @@ class LightningKokkos(LightningQubit):
 
     name = "PennyLane plugin for Kokkos-backed Lightning device"
     short_name = "lightning.kokkos"
-    pennylane_requires = ">=0.22"
+    pennylane_requires = "<=0.28"
     version = __version__
     author = "Xanadu Inc."
     _CPP_BINARY_AVAILABLE = True
+    kokkos_config = {}
```

Comment:
```suggestion
    kokkos_config = kokkos_config_info()
```
Hey @multiphaseCFD. We should be OK to initialize `kokkos_config` directly here. Let me know otherwise.

Reply:
Thanks Amintor! Unfortunately, we couldn't initialize `kokkos_config` directly here since` Kokkos::print_configuration()` can be called only after the `Kokkos` is initialized.

Reply:
That makes sense, thanks for clarifying!